lib/uklibid/Config.uk

```diff
@@ -31,6 +31,14 @@ config LIBUKLIBID_INFO
 				If enabled, the Unikraft release codename is
 				stored as part of the global metadata
 				information.
+
+		config LIBUKLIBID_INFO_UKCONFIGGZ
```

Comment:
🦍 ... nice association 😄 . You mean only the config option or also the record type names: `UKLI_REC_*` and within `support/scripts/uk-libinfo.py`. I kept all in sync and did not use any `_` for the typenames. It is probably personal preference, I don't mind if we do one or the other way. My concern was only that with `_`, some names get really long 🍎 🐍 . Would you also introduce a `_` between `UK` and the type, like: `UK_CONFIG_GZ`, `UK_FULLVERSION`?

Reply:
I think `UK` is kind of an exception because in many places we use `UKXYZ`. But anyway, it you prefer it that way it's also fine, I just find it a bit unusual to the rest of our code.